build/ansible/roles/pmm2-images/tasks/main.yml

```diff
@@ -116,7 +116,7 @@
         non_unique: true
       loop:
         - { name: pmm, uid: 1000, comment: "PMM Server", shell: "/bin/false", home: "/home/pmm", group: pmm, }
-        - { name: nginx, uid: 999, comment: "nginx user", shell: "/sbin/nologin", home: "/var/cache/nginx", group: nginx, }
+        - { name: nginx, uid: 1002, comment: "nginx user", shell: "/sbin/nologin", home: "/var/cache/nginx", group: nginx, }
```

Comment:
Need this because currently nginx conflicts with `systemd-coredump` user and group:
```bash
[root@pmm-server pmm] # id nginx
uid=999(systemd-coredump) gid=997(systemd-coredump) groups=997(systemd-coredump)
```